Use __FILE as marker for file replacement to avoid issues with variables ending in _FILE

SK360 opened the pull request:
Unable to configure HTTPS using ENV overrides due to GF_SERVER_CERT_FILE being picked up as a secret via this code. This change matches one made to the Grafana upstream in 2018.

Fixes #45

SK360 commented:
@googlebot I signed it!

ovk6 commented:
Hi,

Thank you for fixing the script!

Note that our Dockerfiles and entrypoints are being generated from template.
Could you please add respective changes to `templates/docker-entrypoint.sh` file and I will merge PR.

Best Regards

SK360 commented:
> Hi,
> 
> Thank you for fixing the script!
> 
> Note that our Dockerfiles and entrypoints are being generated from template.
> Could you please add respective changes to `templates/docker-entrypoint.sh` file and I will merge PR.
> 
> Best Regards

Doh!  This is what happens when you don't read everything :). I have reverted the previous changes and updated the template.

ovk6 commented:
No no, it was all right :)
Files from templates are generated locally, so PR should contain both templates and target files.

Please un-revert that commit, so you have template and specific versions files
